vtex/utils/transform.ts

```diff
@@ -1120,6 +1121,17 @@ function toHoursSpecification(hours: Hours): OpeningHoursSpecification {
   };
 }
 
+function toSpecialHoursSpecification(
+  holiday: PickupHolidays,
+): OpeningHoursSpecification {
+  return {
+    "@type": "OpeningHoursSpecification",
+    opens: holiday.hourBegin,
+    closes: holiday.hourEnd,
+    validFrom: holiday.date,
+  };
```

Comment:
Do you think it's worth adding "validThrough" so we know when it ends? From what you explained to me, these times are valid for 1 day, perhaps parsing the date that VTEX sends and adding a day will make it clearer for those who are developing and need this data.